improver/ensemble_copula_coupling/utilities.py

```diff
@@ -375,3 +376,60 @@ def interpolate_multiple_rows_same_y(*args):
             "Module numba unavailable. ConvertProbabilitiesToPercentiles will be slower."
         )
         return slow_interp_same_y(*args)
+
+
+def slow_interpolate_pointwise(x, xp, fp):
+    """
+    Given 2 arrays xp and fp with the same dimensions, interpolate along first dimensions
+    at values given by x.
+
+    Args:
+        x: 1-d array, points at which to evaluate interpolation function
+        xp: array with at least 2 dimensions. First dimension is the interpolation dimension.
+        fp: array with at least 2 dimensions. First dimension is the interpolation dimension.
+
+    Returns:
+        array with dimensions (len(x), ) + xp.shape[1:]
+    """
+
+    if xp.shape != fp.shape:
+        raise ValueError("xp and fp must have the same shape")
+    if len(xp.shape) < 2:
+        raise ValueError("xp and fp must have at least 2 dimensions")
```

Comment:
Codecov highlights that these lines aren't covered by any tests. It would be good to add a test to cover these lines and the duplicate lines in `fast_interpolate_pointwise`.

Reply:
Done